app/javascript/react/screens/App/Overview/__test__/__snapshots__/Overview.test.js.snap

```diff
@@ -13,16 +13,26 @@ exports[`Overview component renders the overview 1`] = `
     }
   >
     <Col
+      bsClass="col"
```

Comment:
this is odd... i cleared my `node_modules` and reinstalled a few days ago... and it was changing these snaps...and now they are updating again (guessing we had different dependencies installed). Would anyone be opposed to reintroducing the package lock file? Kind of annoying...

Reply:
The semantic updates to various packages can also cause these...

Reply:
sorry...`yarn.lock` can be added back to prevent this...